integration/Substra/assets/metric/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# this base image works in both CPU and GPU enabled environments
+FROM ghcr.io/substra/substra-tools:0.15.0-nvidiacuda11.6.0-base-ubuntu20.04-python3.9
```

Comment:
Thanks! I corrected this, and I just tried to run in docker but in fact it's not possible (or at least not straightforward) because flamby is a dependency of the metric, but can't be installed with pip. It has to be installed in editable mode to access the datasets...

Reply:
then you can either temporarily copy the source code to the metric folder, then use the `COPY` command to add it to the Docker image, then run `RUN pip install -e .`

or try to git clone the repo from inside the Docker image and install it

And I think you also need to add it to the substrafl algo local dependencies

in all cases it seems like it would take a long time to build and run in Docker mode, maybe we can explain in the README that this example runs in local subprocess mode but not in Docker mode and say a few words why

Reply:
I updated the readme for now. I think there is a problem in setting FLamby as a local dependency  because of the local worker folder.

Reply:
ok, let's solve the local Docker mode and deployed mode in a separate PR